src/datadogConnector/DatadogRumConnector.test.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import DatadogRumConnector from './DatadogRumConnector'
+import { before } from 'node:test'
```

Comment:
I genuinely don't know how this would interact with Jest. 🤔 Let's use the Jest globals instead. We've got [`beforeAll`](https://jestjs.io/docs/api#beforeallfn-timeout) and [`beforeEach`](https://jestjs.io/docs/api#beforeeachfn-timeout) to do what you're looking for.

That aside, I'd actually recommend you change the env vars _in_ the test, then clean up in an [`afterAll`](https://jestjs.io/docs/api#afterallfn-timeout), since these two tests are in the same `describe()` callback. (Without that `afterAll`, side effects may pollute other tests, making them flaky.)